reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -78,38 +103,68 @@
 .heatmapScale .l-6 {
   background-color: #bd0026;
 }
+
+.darkmode .heatmapScale .l-0,
+.darkmode .heatmapScale .l-1,
+.darkmode .heatmapScale .l-2,
+.darkmode .heatmapScale .l-3 {
+  color: #020202;
+}
+
 .heatmapScale .l-7 {
   background-color: #020202;
   color: white;
 }
 
 .heatmapScale {
-  padding-top: 5px;
-  padding-bottom: 5px;
+  padding-top: 0;
+  padding-bottom: 0;
+  margin-bottom: 10px;
+  margin-left: 0;
+  border: 1px solid darkgrey;
 }
-.heatmapScale span {
-  padding: 5px 10px;
+
+.heatmapScale .row {
+  flex-wrap: nowrap;
+}
+
+.darkmode .heatmapScale {
+  border: 1px solid #f2f2f2;
+  background-color: #f2f2f2;
+}
+
+.heatmapScale .col {
+  padding: 2px 15px;
 }
-.heatmapScale span.smallscale {
+.heatmapScale .col.smallscale {
   display: none;
 }
+
 @media all and (max-width: 810px) {
-  .heatmapScale span.smallscale {
+  .heatmapScale .col.smallscale {
     display: inline;
   }
-  .heatmapScale span.largescale {
+  .heatmapScale .col.largescale {
     display: none;
   }
 }
 
+.darkmode .heatmapScale {
```

Comment:
Put these blocks immediately after the blocks they are overriding please

Reply:
✅

Reply:
These blocks still need to be moved

Reply:
refactored